apps/api/src/modules/report/report.controller.ts

```diff
@@ -1,31 +1,39 @@
-import { JobIdDTO } from '@modela/dtos'
+import { JobIdDTO, JwtDto } from '@modela/dtos'
 import { PostReportDTO } from '@modela/dtos'
-import { Body, Controller, Param, Post } from '@nestjs/common'
+import { Body, Controller, Get, Param, Post } from '@nestjs/common'
 import {
   ApiBadRequestResponse,
   ApiCreatedResponse,
   ApiForbiddenResponse,
   ApiNotFoundResponse,
+  ApiOkResponse,
   ApiOperation,
   ApiTags,
   ApiUnauthorizedResponse,
 } from '@nestjs/swagger'
 
+import { UseAuthGuard } from '../auth/misc/jwt.decorator'
+import { User } from '../auth/misc/user.decorator'
 import { ReportService } from './report.service'
 
 @ApiTags('report')
 @Controller('report')
 export class ReportController {
   constructor(private readonly reportService: ReportService) {}
 
-  @Post(':id')
+  @Post('job/:id')
+  @UseAuthGuard()
```

Comment:
Only actor should be able to report? (this endpoint should return 403 for casting and admin)